pyeudiw/satosa/backend.py

```diff
@@ -206,6 +206,8 @@ def pre_request_endpoint(self, context: Context, internal_request, **kwargs) ->
         result = self.template.qrcode_page.render(
             {
                 "qrcode_color": self.config["qrcode"]["color"],
+                "qrcode_size": self.config["qrcode"]["size"],
+                "qrcode_logo_path": self.config["qrcode"]["logo_path"],
                 "qrcode_text": res_url,
```

Comment:
here, pass the static path, configured in the configuration, here:
https://github.com/italia/eudi-wallet-it-python/blob/9487df889a234bd5056040eda8614c1d9945ad5f/example/satosa/pyeudiw_backend.yaml#L7C5-L7C23

and the static storage url MUST be this

`https://localhost/static` 

and then the logo_path must be this

`logo_path: '/wallet-it/wallet-icon-blue.svg'`